core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/SequencerPartitionContainerImpl.java

```diff
@@ -59,15 +71,56 @@
  */
 @Entity
 @Table(name = "SequencerPartitionContainer")
-public class SequencerPartitionContainerImpl extends AbstractSequencerPartitionContainer<SequencerPoolPartition> implements Serializable {
+public class SequencerPartitionContainerImpl implements SequencerPartitionContainer, Serializable {
+
+  private static final long serialVersionUID = 1L;
+  public static final Long UNSAVED_ID = 0L;
+
   private static final Logger log = LoggerFactory.getLogger(SequencerPartitionContainerImpl.class);
 
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long containerId = SequencerPartitionContainerImpl.UNSAVED_ID;
+
+  /**
+   * identificationBarcode is displayed as "serial number" to the user
+   */
+  private String identificationBarcode;
+  private String locationBarcode;
+
+  @ManyToMany(targetEntity = RunImpl.class)
+  @JoinTable(name = "Run_SequencerPartitionContainer", joinColumns = {
+      @JoinColumn(name = "containers_containerId") }, inverseJoinColumns = {
+          @JoinColumn(name = "Run_runId") })
+  private Collection<Run> runs = null;
+
+  @ManyToOne(targetEntity = SecurityProfile.class, cascade = CascadeType.ALL)
```

Comment:
SecurityProfile on PartitionImpl has CascadeType.PERSIST, and here is CascadeType.ALL. It should be the same everywhere